packages/runtime/container-runtime/src/gc/garbageCollection.ts

```diff
@@ -305,6 +307,7 @@ export class GarbageCollector implements IGarbageCollector {
 			eventName: "GarbageCollectorLoaded",
 			gcConfigs: JSON.stringify(this.configs),
 			gcOptions: JSON.stringify(createParams.gcOptions),
+			...createParams.createContainerMetadata,
```

Comment:
Debating about just dumping the whole `createParams.metadata` in here? It will include the `ICreateContainerMetadata` as well (although for new containers it will be blank, that's ok).

A lot of stuff would be functionally redundant with `gcConfigs`, but on the other hand it could be useful to debug if something seems misconfigured - to be able to see plain options and plain metadata as well as computed configs.

Not sure if there's stuff in there that shouldn't be logged though.  Or if the structure includes nested objects.

Reply:
Nested objects is probably what will be problematic. We will have to flatten it out and remove stuff that cannot / shouldn't be logged. We can create a `getLoggableMetadata` and filter things but maybe we can do it incrementally?